capstone/src/main/webapp/statistics_script.js

```diff
@@ -118,42 +123,66 @@ async function getSurveyResponses() {
   loadAgePieChart(ageCount);
 }
 
-function loadBubbleChart(days, scores, genders, responseTimeSThree) {
-  const stats = new google.visualization.DataTable();
+// Default global values
+let directExperience = 'No';
+let gender = 'Male';
+let ageRange = '18-24';
```

Comment:
You usually want your default values to be more abstract than actual values. For example, null or 'Unknown'. It makes it more obvious that these values weren't set.

Reply:
I could set all the values to Unknown at first